.gitignore

```diff
@@ -2,6 +2,7 @@
 *.pyc
 .DS_Store
 /.coverage
+/false/
```

Comment:
What is this for @estaar?

Reply:
My development machine creates that folder everytime am running a virtual environment with pycharm. something to do with Pipenv

Reply:
Is it possibly from npm?

https://github.com/npm/npm/issues/14514

Reply:
I think so. I use Pipenv which has some dependency on node.js for running interactive maps. I will relook at this from my side and see how I can clean it. so in all my .gitignore files am adding this folder.

Reply:
As a workaround you can set up a global "gitignore" for your whole computer: https://stackoverflow.com/a/7335487/1798491.

I would only recommend doing this for files that are specific to your computer and projects are unlikely to have, so looks like this might be a good case.